requirements.txt

```diff
@@ -1,8 +1,7 @@
-colorama
-tiny-tokenizer
```

Comment:
See: https://pypi.org/project/tiny-tokenizer/

> You tried to install “tiny_tokenizer”. The name of tiny_tokenizer renamed to “konoha”

REL didn't use this module explicitly, rather it was a dependency of `flair`. Recent versions of `flair` have updated their dependencies to refer to `konoha` so we should be good.